artifactory/commands/transferfiles/transfer.go

```diff
@@ -12,11 +12,12 @@ import (
 	"github.com/jfrog/jfrog-client-go/utils/errorutils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	"os"
+	"strconv"
 )
 
 const (
 	tasksMaxCapacity = 10000
-	uploadChunkSize  = 100
+	uploadChunkSize  = 10
```

Comment:
After https://github.com/jfrog/data-transfer/pull/12, we can keep it large. Your decision.

Reply:
Agreed.